install/amazon-linux.md

```diff
@@ -0,0 +1,81 @@
+---
+title: Install on Amazon Linux
+---
+
+# Install on Amazon Linux
+
+This document describes how to install PGroonga on Amazon Linux.
+
+## Supported versions
+
+Here are supported Amazon Linux versions:
+
+  * [Amazon Linux 2](#install-on-2)
+
+## How to install on Amazon Linux 2 {#install-on-2}
+
+You can use the following instruction to install PGroonga on Amazon Linux 2.
+
+Install `postgresql10` package:
+
+```text
+% wget https://yum.postgresql.org/10/redhat/rhel-7-x86_64/postgresql10-server-10.9-1PGDG.rhel7.x86_64.rpm
```

Comment:
PostgreSQL version should be variable.

See also:

  * https://github.com/pgroonga/pgroonga.github.io/blob/master/_config.yml#L15
  * https://github.com/pgroonga/pgroonga.github.io/blame/master/install/freebsd.md#L16